src/common/message-creators/user-config-message-creator.ts

```diff
@@ -37,7 +37,7 @@ export class UserConfigMessageCreator extends BaseActionMessageCreator {
         });
     }
 
-    public setBugService(bugServiceName: string): void {
+    public setBugService = (bugServiceName: string) => {
```

Comment:
This changes looks odd, care to elaborate? 
Also, I have 2 or 3 PR's to refactor some action message creators to use the new action message dispatchers so this file will change any way?

Reply:
This is to avoid using autobind; allows proper scoping for 'this'.

Reply:
Nice... didn't think of that.